src/index.js

```diff
@@ -62,7 +62,7 @@ todoArray.forEach((todo) => {
     if (todoListCheck.checked === true) {
       todoListCheck.nextElementSibling.style.textDecoration = 'line-through';
       todoListCheck.nextElementSibling.style.opacity = '0.5';
```

Comment:
you can style your completed to do element by adding and removing a class with specific style instead for writing it in multiple places

Reply:
I resloved it too!